build.sbt

```diff
@@ -261,6 +261,9 @@ lazy val root = (project in file("."))
   )
   .settings(
       testSettings,
+      // TODO: uncomment to enable building unidoc for for all test and src code
+      // Generate scaladoc for both test and src code
+      // ScalaUnidoc / unidoc / unidocConfigurationFilter := inConfigurations(Compile, Test),
```

Comment:
This will configure unidoc to build scaladoc for all our tests and the source code (assuming this what we want). But there are many failures in the tests, so I'd like to first get the scaladoc build for our src into CI to prevent additional breakage from being added, and then work on cleaning up the problems in the tests.